worldedit-core/src/main/java/com/sk89q/worldedit/WorldEditException.java

```diff
@@ -34,19 +42,50 @@ protected WorldEditException() {
      * Create a new exception with a message.
      *
      * @param message the message
+     * @deprecated Use component version
      */
+    @Deprecated
     protected WorldEditException(String message) {
         super(message);
+
+        this.message = TextComponent.of(message);
+    }
+
+    /**
+     * Create a new exception with a message.
+     *
+     * @param message the message
+     */
+    protected WorldEditException(Component message) {
+        super(WorldEditText.reduceToText(message, Locale.US));
```

Comment:
Wonder if we should do this always in `US` or in the server's default locale. It might be more useful for server admins to debug if the language is what they expect, even if it's a little harder for us to debug.